src/presenter/point-new-presenter.js

```diff
@@ -0,0 +1,82 @@
+import dayjs from 'dayjs';
+import {nanoid} from 'nanoid';
+import FormEditView from '../view/form-edit-view.js';
+import {render, RenderPosition, remove} from '../utils/render.js';
+import {UserAction, UpdateType} from '../consts/common.js';
+
+const BLANK_OFFER = {
+  'basePrice': '',
+  'dateFrom': dayjs().toDate(),
+  'dateTo': dayjs().toDate(),
+  'id': nanoid(5),
+  'isFavorite': '',
+  'type': '',
+  'destination': {
+    'description': '',
+    'name': '',
+  },
+  'offers': [],
+};
+
+export default class PointNewPresenter {
+  #pointListContainer = null;
+
+  #changeData = null;
+
+  #pointEditComponent = null;
+
+  #point = null;
+  #formType = '';
```

Comment:
а может вот так писать? 
```suggestion
  #pointListContainer = null;
  #changeData = null;
  #pointEditComponent = null;
  #point = null;
  #formType = '';
```